ee/clickhouse/queries/funnels/test/test_funnel_correlation.py

```diff
@@ -160,6 +171,104 @@ def test_basic_funnel_correlation_with_events(self):
         self.assertEqual(len(self._get_people_for_event(filter, "negatively_related", success=False)), 5)
         self.assertEqual(len(self._get_people_for_event(filter, "negatively_related")), 0)
 
+    @snapshot_clickhouse_queries
+    def test_funnel_correlation_with_events_and_actions(self):
+
+        for i in range(10):
+            _create_person(distinct_ids=[f"user_{i}"], team_id=self.team.pk)
+            _create_event(
```

Comment:
non-blocking as is consistent with other tests in the file.

Can we use the `ee.clickhouse.test.test_journeys.journeys_for` to construct these scenarios?